crates/sui-protocol-config/src/lib.rs

```diff
@@ -1163,6 +1166,12 @@ pub struct ProtocolConfig {
     // Note: this is not a feature flag because we want to distinguish between
     // `None` and `Some(false)`, as committee was already finalized on Testnet.
     bridge_should_try_to_finalize_committee: Option<bool>,
+
+    /// The max accumulated txn execution cost per object in a mysticeti. Transactions
+    /// in a commit will be deferred once their touch shared objects hit this limit.
+    /// This config plays the same role as `max_accumulated_txn_cost_per_object_in_narwhal_commit`
+    /// but for mysticeti commits due to that mysticeti has higher commit rate.
+    max_accumulated_txn_cost_per_object_in_mysticeti_commit: Option<u64>,
```

Comment:
nit: maybe move this next to `max_accumulated_txn_cost_per_object_in_narwhal_commit`?

Reply:
Not sure if we can change the order, since this is also persisted in DB: https://docs.rs/bcs/latest/bcs/#structures

Reply:
Just curious, where is this persisted in DB? I know there is ongoing work to BCS serialize feature flags or protocol config but not sure if that is already the case.

Reply:
Yeah, you are right. I think I'm confused it with `EpochStartConfiguration`. The order indeed does not matter, and there are recent configs added in the middle of the struct. Let me move them together.